test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3666,6 +3666,42 @@ public void testvaltesttStopOnFirstFailedTestCase() throws Exception {
         }
         
     }
+    
+    
+    /**
+     * Test yaml import for memory-limit-in-Meg and sandbox.
+     * 
+     * @throws Exception
+     */
+    
+    public void testLodSandboxAndMemoryLimit() throws Exception {
+        
```

Comment:
Change name of method: "Lod" should be "Load"

Reply:
Fixing typo now.